system/services/email/SystemEmailTemplateService.cfc

```diff
@@ -26,25 +26,77 @@ component {
 	 *
 	 */
 	public array function listTemplates() {
-		var templateIds = _getConfiguredTemplates().keyArray();
-		var templates   = [];
+		if ( isArray( __templateList ?: "" ) ) {
```

Comment:
I don't think these can be cached, _unless_ we add the current request language into the cache key (as translateresource is request specific)